rwengine/src/audio/SoundManager.cpp

```diff
@@ -87,6 +92,22 @@ void SoundManager::initializeAVCodec() {
 #endif
 }
 
+bool SoundManager::initializeEFX() {
+    if (!alcIsExtensionPresent(alDevice, ALC_EXT_EFX_NAME)) {
+        RW_ERROR("EFX extention is not present");
+        return false;
+    }
+
+    initEfxFunctionPointers();
+
+    // first effect is placeholder
+    // it allows index to be like in enum
+    soundEffects.push_back(nullptr);
```

Comment:
Maybe it's better to remove `None` from enum and add `bool` enabled. I think there are too many parameters for sfx to pass them around. Probably we should add some abstraction/wrapper later.